SECURITY.md

```diff
@@ -0,0 +1,42 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version       | Supported          |
+| -------       | ------------------ |
+| v0.1.5        |      Yes           |
+| v0.1.0-alpha  | Alpha Testing      |
+
+
+## Reporting a Vulnerability
+
+You will receive a response from us within 2 working days. If the issue is confirmed, we will release a patch as soon as possible depending on complexity but historically within a few days.
+
+Please avoid opening public issues on GitHub that contain information about a potential security vulnerability as this makes it difficult to reduce the impact and harm of valid security issues.
+
+## Coordinated Vulnerability Disclosure Policy
+
+We ask security researchers to keep vulnerabilities and communications around vulnerability submissions private and confidential until a patch is developed. In addition to this, we ask that you:
+
+- Allow us a reasonable amount of time to correct or address security vulnerabilities.
+- Avoid exploiting any vulnerabilities that you discover.
+- Demonstrate good faith by not disrupting or degrading services built on top of this software.
+
+## Vulnerability Disclosure Process
+
+Confio uses the following disclosure process for the various CosmWasm-related repos:
```

Comment:
I believe that here we should change our context

Reply:
Please let me know how you would like to add the changes too ?

Reply:
"**Confio** uses the following disclosure process for the various **CosmWasm**-related repos:"

I believe you used the SECURITY.md from CosmOsm/Confio as a base.

We just need to change this info (CosmOsm and CONFIO) to Octobear

Reply:
Updated !!